### PR TITLE
MOYO navbar created

### PR DESCRIPTION
**Pull request by gnosis-frog**
- [DEMO LINK](https://gnosis-frog.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://gnosis-frog.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by Elena-brv — CHANGES_REQUESTED** [review]
- this indent shouldn't be here
image

- checklist about links

File: src/style.css [hunk 1]
```diff
@@ -1 +1,48 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
```
Comment (Elena-brv) [inline src/style.css:1]: add fonts via a link in the `<head>`

File: src/style.css [hunk 2]
```diff
@@ -1 +1,48 @@
...
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  text-transform: uppercase;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+  float: left;
```
Comment (Elena-brv) [inline src/style.css:16]: use display: flex for .header instead of this property

**Review by Elena-brv — CHANGES_REQUESTED** [review]
- indents should be the same 
image

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**